services/_lambda.py

```diff
@@ -1,29 +1,95 @@
 from models import RuleCheckResult
 import boto3
+import json
 
 
-# client = boto3.client("")
+client = boto3.client("lambda")
+iam_client = boto3.client("iam")
 
 
 def lambda_dlq_check():
+    compliant_resource = []
+    non_compliant_resources = []
+    functions = client.list_functions()["Functions"]
+
+    for function in functions:
+        response = client.get_function(FunctionName=function["FunctionName"])["Configuration"]
+        if "DeadLetterConfig" in response:
```

Comment:
`get_function`을 한번 더 호출하지 않아도 동일하게 설정 값을 확인할 수 있습니다.
```suggestion
    functions = client.list_functions()["Functions"]

    for function in functions:
        if "DeadLetterConfig" in function:
```